tests/test_architecture.py

```diff
@@ -0,0 +1,130 @@
+from pytest_archon import archrule
+
+import gaphor
+from gaphor.entrypoint import load_entry_points
+
+GAPHOR_CORE = [
+    "gaphor.core*",
+    "gaphor.abc",
+    "gaphor.action",
+    "gaphor.entrypoint",
+    "gaphor.i18n",
+    "gaphor.transaction",
+    "gaphor.event",
+]
+
+UI_LIBRARIES = [
+    "gi.repository.Gdk",
+    "gi.repository.Gtk",
```

Comment:
Should we add gi.repository.Adw to the UI libraries?

Reply:
Can do that, just to be on the safe side.